src/chain/member_info.rs

```diff
@@ -7,18 +7,40 @@
 // permissions and limitations relating to use of the SAFE Network Software.
 
 /// Information about a member of our section.
+
+pub type AgeCounter = u32;
+
+/// The minimum allowed value of the Age Counter
+pub const MIN_AGE: AgeCounter = 16;
```

Comment:
Why 16?

Reply:
It's pretty arbitrary, but we decided that the nodes' starting age should be 4, which corresponds to the counter value of 16. I could call it `STARTING_AGE` if that makes more sense, or `STARTING_AGE_COUNTER`/`MIN_AGE_COUNTER`.

Reply:
Probably should be `MIN_AGE_COUNTER` or `STARTING_AGE_COUNTER`. 

> It's pretty arbitrary, but we decided that the nodes' starting age should be 4
Any particular reason? I would expect the age to start at 0 or 1. My guess it most people would.

Reply:
The reason was not to relocate infants too much - since we are planning on relocating nodes every time their age increases (not age counter, just age), they would get relocated almost immediately if the age started at 0 or 1. This delays the first relocation to after 16 churn events, which is not immediately and not too much at the same time.

We could redefine the age to be `log2(counter) - 4`, for example, but I'm not sure if it's worth it.

Reply:
OK, that makes sense. Thanks for the explanation. Maybe worth adding a comment with this info too?